roles/devscripts/tasks/133_host_network.yml

```diff
@@ -36,14 +36,85 @@
     name: ci_network
     tasks_from: apply-dns.yml
 
-- name: Ensure virtual networks are created.
+- name: Ensure the required virtual networks exist.
+  become: true
   when:
     - cifmw_devscripts_config.manage_br_bridge is defined
     - cifmw_devscripts_config.manage_br_bridge == 'n'
   vars:
-    cifmw_libvirt_manager_net_prefix_add: false
-    _layout:
-      networks: "{{ cifmw_libvirt_manager_configuration.networks }}"
-  ansible.builtin.include_role:
-    name: libvirt_manager
-    tasks_from: create_networks.yml
+    _nets: >-
+      {{
+        cifmw_libvirt_manager_configuration.networks |
+        dict2items |
+        map(attribute='key') |
+        list
+      }}
+  block:
+    - name: Ensure br_netfilter module is loaded.
+      community.general.modprobe:
+        name: br_netfilter
+        state: present
+
+    - name: Ensure IP forwarding is enabled.
+      ansible.posix.sysctl:
+        name: net.ipv4.ip_forward
+        value: 1
+        state: present
+
+    - name: Ensure the required parameters are loaded.
+      ansible.posix.sysctl:
+        name: "net.bridge.bridge-nf-call-{{ item }}"
+        value: 0
+        state: present
+      loop:
+        - arptables
+        - iptables
+        - ip6tables
+      loop_control:
+        label: "{{ item }}"
+
+    - name: Ensure firewalld service is enabled and started.
```

Comment:
hmmmm isn't it weird to start `firewalld` service but then use `iptables` to inject rules? I'd rather use https://docs.ansible.com/ansible/latest/collections/ansible/posix/firewalld_module.html for the sake of consistency.

Any thoughts?

Reply:
I had tried the steps outlined here https://docs.fedoraproject.org/en-US/fedora-server/administration/dnsmasq/#_masquerading_nat
with no success. (without bridge-nf changes)

Let me revisit it

Reply:
Sadly, `firewalld` module is limited on my requirements however I am evaluating a shell script that uses the `firewall-cmd` 

I understand that is not ansible like... open to suggestions though.

Reply:
Bleh... in that case, maybe sticking to "pure" ansible and, therefore, iptables module, may be OK. I'm just not that confident mixing multiple tools managing the same kind of resources is a good idea :(.

Do we have other places where a firewall is managed? Just trying to see if there's already a base for either iptables or firewalld - or, heh, nftables (the iptables replacement - `iptables` CLI calls nftables under the hood).

Reply:
Shifted to the older approach.